packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/upgrade-guides.5/version-3-upgrade-guide.3.tool.md

```diff
@@ -0,0 +1,7 @@
+# Terra Functional Testing - Version 3 Upgrade Guide
```

Comment:
Can you add a changelog entry in terra-toolkit-docs for this? There is no failure because no build occurred. This begs the question that we should add the toolkit-mvb branch in .travis.yml to kick off a build when merging into this branch.

See https://github.com/cerner/terra-core/blob/main/.travis.yml#L8

Reply:
Added changelog entry for Toolkit-Docs : [f327f90](https://github.com/cerner/terra-toolkit/pull/741/commits/f327f90807bd9fd415920b7b093a31adb60e1351)